packages/notifi-react-card/lib/hooks/SubscriptionCardConfig.ts

```diff
@@ -1,4 +1,5 @@
 // TODO: Import from library rather than copy / paste
+import { AlertFrequency } from 'notifi-frontend-client/dist';
```

Comment:
this is an incorrect import~ import from `@notifi-network/notifi-core` instead

btw, any time you see a `/dist` import, that's not right. It's due to Visual Studio using relative imports.

Reply:
ah, good catch! updated